queue/driver_locker.go

```diff
@@ -98,7 +98,29 @@ func (l *lockManager) lockPinger(ctx context.Context) {
 				}
 
 				stat := j.Status()
-				if !stat.InProgress || stat.Owner != l.name {
+				if !stat.InProgress {
```

Comment:
is `!state.InProgress` really the same thing as a complete job? I don't know what the other state are.

Reply:
we have a job, we lock it (e.g. set it in progress) and then we put it in the ping list, which is what this list is. 

We should keep pinging it until it's no longer locked, (e.g. not in progress) regardless of if it was marked complete or not, so I'm pretty sure this is correct.